src/__experimental__/components/textbox/documentation/info.js

```diff
@@ -8,7 +8,7 @@ const info = (
 
     <StoryHeader>Implementation</StoryHeader>
 
-    <StoryCode padded>import Textbox from {'carbon-react/lib/components/textbox'}</StoryCode>
+    <StoryCode padded>import Textbox from carbon-react/lib/components/textbox</StoryCode>
```

Comment:
no quotes here?

Reply:
fixed in https://github.com/Sage/carbon/commit/abb7610908dcd5f712bb0326ce4e2f2ae264e67f